releasenotes/notes/add-linear-ancilla-composite-3281ed6733b0f0c7.yaml

```
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Add `LinearAncillaComposite` for implementing linear biases through ancilla qubits 
```

```suggestion
    Add `LinearAncillaComposite` for emulating linear coefficients through flux-bias offsets on ancilla qubits. 
```

See [this system doc section](https://docs.dwavesys.com/docs/latest/handbook_qpu.html#emulate-linear-biases-with-flux-bias-offsets) for the terminology we're using

---

Ha. First time I see this system doc section